frontend/src/components/calendar/Calendar.js

```diff
@@ -49,85 +58,270 @@ const eventUser = {
       </div>
     ),
   },
-}
+};
 
 class Calendar extends React.Component {
```

Comment:
Not that this should be changed, but why is a class component being used here?

Reply:
I feel more comfortable writing larger components as class components. Just a stylistic choice I made at the very beginning. I sort of like larger, smart components to be class-based since I think the functions used make more sense.